recipes-ni/initscripts-nilrt/files/nisetupkernelconfig

```diff
@@ -7,6 +7,9 @@
 #executable name as the suffix.
 safemodeflag=/etc/natinst/safemode
 if [ -f $safemodeflag ]; then
+  if [ ! -d /mnt/userfs/var/local/natinst/log ]; then
```

Comment:
This creates the directory for admin user. After a runmode is installed, this directory will be recreated with lvuser permission. Core dump works regardless of the permission.

Reply:
Is the directory and it's contents cleared and created again when runmode is installed?

Reply:
Yes